maat-court-data-api/src/main/java/gov/uk/courtdata/entity/RepOrderEquityEntity.java

```diff
@@ -0,0 +1,88 @@
+package gov.uk.courtdata.entity;
+
+import jakarta.annotation.Nullable;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.*;
+
+import jakarta.persistence.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Data
+@Table(name = "REP_ORDER_EQUITY", schema = "TOGDATA")
+public class RepOrderEquityEntity {
+
+    @Id
+    @Column(name = "ID", nullable = false)
+    @NotNull
+    @SequenceGenerator(name = "rep_order_equity_seq", sequenceName = "S_GENERAL_SEQUENCE", allocationSize = 1, schema = "TOGDATA")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "rep_order_equity_seq")
+    private Integer id;
+
+    @NotNull
+    @Column(name = "REP_ID")
+    private Integer repId;
+
+    @Nullable
+    @Column(name = "ROPD_ID")
+    private Integer ropdId;
```

Comment:
Do we know what ropd stands for? Wonder if we can make our variable name more descriptive even if the column name is not. :)